docs/model/how-to-work-with-money.md

```diff
@@ -40,12 +40,12 @@ The scale has to be specified during [rounding](#rounding), [creating from float
 
 ## Money Class
 
-[`Money`](/packages/framework/src/Component/Money/Money.php) is an immutable [value object](https://codete.com/blog/value-objects/).
+[`Money`](/Component/Money/Money.php) is an immutable [value object](https://codete.com/blog/value-objects/).
```

Comment:
All path changes in this file in d615476a4572b00726a31bcfea38c308548c19d0 broke the links.

Reply:
fixed